weareinreach/inreach-catalog
# Feature/new account login disclaimer

## Description



- Asana ticket: https://app.asana.com/0/1132189118126148/1166444744229705?trial=true

## PR Checklist




## How to Test

- Open the login form by clicking "log in" at the top of the page
- Verify that the banner appears where it should and that the spelling is correct
